gcp_variant_transforms/libs/processed_variant.py

```diff
@@ -420,15 +452,25 @@ def _create_map(self, annotations, annotation_names):
       raise ValueError('Expected {} annotations, got {}'.format(
           len(annotation_names), len(annotations) - 1))
     annotation_dict = {}
+    annotation_dict[_ANNOTATION_ALT] = annotations[0]
     for index, name in enumerate(annotation_names):
       annotation_dict[name] = annotations[index + 1]
     return annotation_dict
 
-  def _add_annotation_list(
-      self, proc_var, common_prefix, alt_bases, annotations_list,
-      annotation_field_name):
-    # type: (ProcessedVariant, str, str, List[Dict[str, str]], str) -> None
-    """Adds all annotations to the given `proc_var`.
+  def _add_ambiguous_fields(self, annotations_list, ambiguous):
+    # type: (List[Dict[str, str]], bool) -> None
+    if self._minimal_match:
```

Comment:
nit: i think it's more readable to move this`if` at the top so that we don't call this method if minimal_match is false. In general, methods should perform what they are meant to do and not become a no-op based on external factors.

Reply:
Done.